docker-compose/technical/create-postgres-databases.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+function createDatabases()
+{
+psql --username postgres --dbname postgres <<-EOSQL
```

Comment:
Do we have to put username (maybe password?) and dbname into .env file and use variable here ?

Reply:
Done